Makefile

```diff
@@ -154,41 +196,90 @@ $(PROTO_MODEL): $(OUTPUT_DIR)/unicharset data/radical-stroke.txt
 	combine_lang_model \
 	  --input_unicharset $(OUTPUT_DIR)/unicharset \
 	  --script_dir data \
+	  --numbers $(NUMBERS_FILE) \
+	  --puncs $(PUNC_FILE) \
+	  --words $(WORDLIST_FILE) \
 	  --output_dir data \
+	  $(RECODER) \
 	  --lang $(MODEL_NAME)
 
 ifdef START_MODEL
+ifeq ($(BUILD_TYPE),Impact)
+$(LAST_CHECKPOINT): unicharset lists
+	@mkdir -p $(OUTPUT_DIR)/checkpoints
+	lstmtraining \
+	  --debug_interval $(DEBUG_INTERVAL) \
+	  --traineddata $(TESSDATA)/$(START_MODEL).traineddata \
+	  --continue_from data/$(START_MODEL)/$(MODEL_NAME).lstm \
+	  --model_output $(OUTPUT_DIR)/checkpoints/$(MODEL_NAME)$(BUILD_TYPE) \
+	  --train_listfile $(OUTPUT_DIR)/list.train \
+	  --max_iterations $(MAX_ITERATIONS)
+$(OUTPUT_DIR)$(BUILD_TYPE).traineddata: $(LAST_CHECKPOINT)
+	lstmtraining \
+	--stop_training \
+	--continue_from $(LAST_CHECKPOINT) \
+	--traineddata $(TESSDATA)/$(START_MODEL).traineddata \
+	--model_output $@
+endif
+ifeq ($(BUILD_TYPE),Plus)
 $(LAST_CHECKPOINT): unicharset lists $(PROTO_MODEL)
 	@mkdir -p $(OUTPUT_DIR)/checkpoints
 	lstmtraining \
+	  --debug_interval $(DEBUG_INTERVAL) \
 	  --traineddata $(PROTO_MODEL) \
 	  --old_traineddata $(TESSDATA)/$(START_MODEL).traineddata \
-	  --continue_from data/$(START_MODEL)/$(START_MODEL).lstm \
-	  --net_spec "$(subst c###,c`head -n1 $(OUTPUT_DIR)/unicharset`,$(NET_SPEC))" \
```

Comment:
I am lost. Why don't you need this parameter anymore?

Reply:
Do you implicitly use the net specification from the start model?

Reply:
https://github.com/tesseract-ocr/tesseract/wiki/TrainingTesseract-4.00#fine-tuning-for--a-few-characters

```
training/lstmtraining --model_output /path/to/output [--max_image_MB 6000] \
  --continue_from /path/to/existing/model \
  --traineddata /path/to/traineddata/with/new/unicharset \
  --old_traineddata /path/to/existing/traineddata \
  [--perfect_sample_delay 0] [--debug_interval 0] \
  [--max_iterations 0] [--target_error_rate 0.01] \
  --train_listfile /path/to/list/of/filenames.txt
```

Reply:
I have removed the options for IMPACT and PLUS training.

If START_MODEL is given then it will be treated as replace a LAYER training - https://github.com/tesseract-ocr/tesseract/wiki/TrainingTesseract-4.00#training-just-a-few-layers

The following is the tutorial example:

```
mkdir -p ~/tesstutorial/eng_from_chi
training/combine_tessdata -e tessdata/best/chi_sim.traineddata \
  ~/tesstutorial/eng_from_chi/eng.lstm
training/lstmtraining --debug_interval 100 \
  --continue_from ~/tesstutorial/eng_from_chi/eng.lstm \
  --traineddata ~/tesstutorial/engtrain/eng/eng.traineddata \
  --append_index 5 --net_spec '[Lfx256 O1c111]' \
  --model_output ~/tesstutorial/eng_from_chi/base \
  --train_listfile ~/tesstutorial/engtrain/eng.training_files.txt \
  --eval_listfile ~/tesstutorial/engeval/eng.training_files.txt \
  --max_iterations 3000 &>~/tesstutorial/eng_from_chi/basetrain.log
```

Reply:
>Why don't you need this parameter anymore?

AFAIK net_spec in the full form is only needed for training from scratch.

Unless, there is an undocumented way in which it is used with start_model.

Reply:
This was a misunderstanding. My apologies @Shreeshrii ! I was mislead by the syntax highlighting.

Reply:
No problem, @wrznr. I am going to  remove the changes for finetune options from this PR to limit it just for language types.

Please let me know whether I should provide testdata for each language type to make it easier to review.